lib/dynflow/config.rb

```diff
@@ -98,13 +98,21 @@ def queues
     end
 
     config_attr :executor, Executors::Abstract, FalseClass do |world, config|
-      Executors::Parallel.new(world, config.queues)
+      Executors::Parallel.new(world, config.executor_heartbeat_interval, config.queues)
```

Comment:
This needs to be updated also here https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/rails/configuration.rb#L158, otherwise the rails integration breaks.